tests/common.go

```diff
@@ -7719,7 +7719,8 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		log.FailOnError(err, "failed to switch context to source cluster")
 	}()
 	masterNodes := node.GetMasterNodes()
```

Comment:
@sn-px , Can you remove the above defer block at line 7717 in this PR.
Currently its trying to fetch the px-backup logs from destination cluster.

please find the below error:
https://aetos.pwx.purestorage.com/resultSet/testSetID/590948/testCaseID/3624513

Reply:
done